spec/features/applications/show_spec.rb

```diff
@@ -24,5 +24,35 @@
         expect(page).to have_content(@application1.status)
       end
     end
+
+    describe 'add pets to application' do
+      it 'has a way to add pets to an application' do
+        visit "/applications/#{@application1.id}"
+
+        expect(page).to have_content('Add a Pet to this Application')
+      end
+
+      it 'can search for a pet' do
+        visit "/applications/#{@application1.id}"
+
+        fill_in 'pet_name', with: 'Buddy'
+
+        click_button 'Submit'
+
+        expect(page).to have_content('Buddy')
+      end
+
+      it 'has a button and can add each pet to application' do
```

Comment:
I think these tests work well as unit tests. Maybe we should look into unit feature tests in the future.